packages/codemods/src/v8-v9/transforms/__snapshots__/chrome.spec.ts.snap

```diff
@@ -0,0 +1,97 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I would not typically recommend snapshot testing; however, among all the available unit testing [methods](https://github.com/facebook/jscodeshift/blob/main/README.md#unit-testing), `defineSnapshotTest` appears to be the easiest to set up while still ensuring that the test files remain reasonably readable.